mob.go

```diff
@@ -310,27 +336,16 @@ func removeWipPrefix(branch string) string { //TODO improve, add tests
 	return branch[len(wipBranchPrefix):]
 }
 
-func addSuffix(branch string) string { // TODO rename to addWipQualifier
-	return branch + configuration.WipBranchQualifierSeparator + configuration.WipBranchQualifier
+func addSuffix(branch string, suffix string) string {
+	return branch + suffix
 }
 
-func hasSuffix(branch string) bool { //TODO improve (dont use strings.Contains, add tests)
+func hasWipBranchQualifierSeparator(branch string, configuration Configuration) bool { //TODO improve (dont use strings.Contains, add tests)
 	return strings.Contains(branch, configuration.WipBranchQualifierSeparator)
 }
 
-func removeSuffix(branch string) string {
-	if configuration.WipBranchQualifierSet { // WipBranchQualifier configured
-		suffix := configuration.WipBranchQualifierSeparator + configuration.WipBranchQualifier
-		if strings.HasSuffix(branch, suffix) {
-			return branch[:strings.LastIndex(branch, suffix)]
-		} else {
-			return branch
-		}
-	} else if hasSuffix(branch) { // WipBranchQualifier not configured, but WipBranchQualifierSeparator found
-		return branch[:strings.LastIndex(branch, configuration.WipBranchQualifierSeparator)]
-	} else {
-		return branch
-	}
+func wipBranchQualifierSuffix(configuration Configuration) string {
```

Comment:
Such functions could become methods of the configuration struct.

Reply:
made it so